src/pages/rechner/report/[responseId].tsx

```diff
@@ -37,19 +65,42 @@ const Report: NextPage<Props> = ({ impactInTons, fte, year }) => (
               </div>
               <div className="p-6">
                 <span className="text-xxs">
-                  Pro Vollzeitmitarbeiter: {Math.round(((impact / fte) * 100) / 100)} t CO<sub>2</sub>
+                  Pro Vollzeitmitarbeiter ca.: {Math.round(((impact / fte) * 1000) / 1000)} t CO<sub>2</sub>
                 </span>
                 <Heading2>{name}</Heading2>
-                <span>Was passiert hier?</span>
               </div>
             </div>
           );
         })}
       </div>
+      <br /> <br /> <br /> <br /> <br />
```

Comment:
On the previous div you already work with padding (`pb-16`). why not increase it to create this space instead of using all these `<br />`? Or even better use margin which is supposed to be used for spacing around containers.